.github/workflows/convert_to_python_script.yml

```diff
@@ -11,8 +11,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-        with:
-          ref: ${{ github.head_ref }}
```

Comment:
Se essa modificação está funcionando aqui temos que abrir um novo PR com essa mudança e, depois que for mergeada, atualizar as demais branches.